<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPBYMINW2706" ignored="0" name="Cosine Cal Test" tests="4" failures="4" timestamp="28 May 2019 12:19:20 GMT" time="1.402" errors="0">
  <testcase name="cosTest" time="0.0" classname="com.epam.cdp.cal.tests.CosineCalculatorTest">
    <failure type="java.lang.AssertionError" message="Invalid result of cosine expected [-0.9055783620066238] but found [0.42]">
      <![CDATA[java.lang.AssertionError: Invalid result of cosine expected [-0.9055783620066238] but found [0.42]
at com.epam.cdp.cal.tests.CosineCalculatorTest.cosTest(CosineCalculatorTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- cosTest -->
  <testcase name="cosTest" time="0.0" classname="com.epam.cdp.cal.tests.CosineCalculatorTest">
    <failure type="java.lang.AssertionError" message="Invalid result of cosine expected [-0.9613974918795568] but found [-0.275]">
      <![CDATA[java.lang.AssertionError: Invalid result of cosine expected [-0.9613974918795568] but found [-0.275]
at com.epam.cdp.cal.tests.CosineCalculatorTest.cosTest(CosineCalculatorTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- cosTest -->
  <testcase name="cosTest" time="0.0" classname="com.epam.cdp.cal.tests.CosineCalculatorTest">
    <failure type="java.lang.AssertionError" message="Invalid result of cosine expected [0.1411200080598672] but found [-0.98]">
      <![CDATA[java.lang.AssertionError: Invalid result of cosine expected [0.1411200080598672] but found [-0.98]
at com.epam.cdp.cal.tests.CosineCalculatorTest.cosTest(CosineCalculatorTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- cosTest -->
  <testcase name="cosTest" time="0.0" classname="com.epam.cdp.cal.tests.CosineCalculatorTest">
    <failure type="java.lang.AssertionError" message="Invalid result of cosine expected [0.8414709848078965] but found [0.54]">
      <![CDATA[java.lang.AssertionError: Invalid result of cosine expected [0.8414709848078965] but found [0.54]
at com.epam.cdp.cal.tests.CosineCalculatorTest.cosTest(CosineCalculatorTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- cosTest -->
</testsuite> <!-- Cosine Cal Test -->
